azure-pipelines.yml

```diff
@@ -90,9 +90,10 @@ jobs:
           SourcePath: 'apps/pr-deploy-site/dist'
           storage: $(azureStorage)
 
-      # - script: |
-      #     yarn e2e $(sinceArg)
-      #   displayName: Cypress E2E tests
+      # only run e2e tests when converged storybook is published by scoping to the converged suite package
+      - script: |
+          yarn e2e $(sinceArg) --scope @fluentui/react-components
```

Comment:
Q: this is where this magic variable comes from ?

![image](https://user-images.githubusercontent.com/1223799/117002307-afaf7f80-ace3-11eb-8f8d-34e52dd8d444.png)

Reply:
I think our screenshot answers the question ? it's a variable declared in the file

Reply:
yeah just wanted to double check, thx. not a fan of imperative logic via magic strings

Reply:
This was to avoid having to duplicate several different steps with different conditions for PR and master builds.